src/config/wmodules-aws.c

```diff
@@ -59,6 +59,7 @@ static const char *INSPECTOR_SERVICE_TYPE = "inspector";
 static const char *CLOUDWATCHLOGS_SERVICE_TYPE = "cloudwatchlogs";
 static const char *CISCO_UMBRELLA_BUCKET_TYPE = "cisco_umbrella";
 static const char *SECURITY_LAKE_SUBSCRIBER_TYPE = "security_lake";
+static const char *BUCKETS_SUBSCRIBER_TYPE = "buckets";
```

Comment:
Why not in singular?

Reply:
The reason for the type `buckets` was described in [this update](https://github.com/wazuh/wazuh/issues/13577#issuecomment-1638715385).